kumascript/tests/macros/mdnsidebar.test.js

```diff
@@ -7,10 +7,10 @@ const jsdom = require("jsdom");
 
 const locales = {
   "en-US": {
-    About_MDN: "About MDN",
+    About_MDN: "Contributing to MDN",
```

Comment:
Shouldn't that be 
```suggestion
    Contributing_to_MDN: "Contributing to MDN",
```
and same for French below.